impl/client/src/main/doc-js/pentaho/type/spec/InstanceReservation.jsdoc

```diff
@@ -0,0 +1,94 @@
+/*!
+ * Copyright 2017 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The `InstanceReservation` enumeration contains the possible reservation modes
+ * that a synchronous instance resolution operation can use.
+ *
```

Comment:
Should we list the reservation modes here for quickly knowing the possible modes at the beginning of the doc?

Reply:
This is a JsDocs enum. It is already formatted as a table of values.